src/components/templates/ReactionPage/ReactionList.tsx

```diff
@@ -71,11 +72,11 @@ export const ReactionList: React.FC<ReactionListProps> = ({
 
           <div
             className={classNames("message-bubble", {
-              emoji: message.reaction !== "messageToTheBand",
+              emoji: isEmojiReaction(message),
             })}
           >
-            {message.reaction !== "messageToTheBand"
-              ? ReactionsTextMap[message.reaction]
+            {isEmojiReaction(message)
+              ? EmojiReactionsMap.get(message.reaction)?.text ?? null
```

Comment:
Why do we need `?? null`?

Reply:
I believe I was basing it on the following:

- https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

I wasn't sure that `undefined` was a valid usage for this:

- https://github.com/facebook/react/issues/1647#issuecomment-64447594

But it seems like it's fine:

- https://stackoverflow.com/questions/42083181/is-it-possible-to-return-empty-in-react-render-function/42083262#42083262
  - https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

So we can simplify the code here.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/e8528100e2e3e41174a0e2af043797a858dbf317